Rakefile

```
@@ -2,7 +2,7 @@ require 'rubygems/package_task'
 require 'rake/extensiontask'
 require 'rake/testtask'
 
-task default: [:compile, :test]
+task default: %w[test:pure_ruby test:compiled]
```

The PR looks good but will cause issues to users who use `rake test` for their task. I would suggest add a `test` task for backward compatibility.

```suggestion
task default: :test
task test: %w[test:pure_ruby test:compiled]
```